src/sidebar/test/annotation-ui-test.js

```diff
@@ -79,6 +80,27 @@ describe('annotationUI', function () {
         [sinon.match(annot)]);
     });
 
+    it('does not change selectedTab state if annotations are already loaded', function () {
+      var annot = defaultAnnotation();
+      annotationUI.addAnnotations([annot]);
+      var page = oldPageNote();
+      annotationUI.addAnnotations([page]);
+      assert.equal(annotationUI.getState().selectedTab, uiConstants.TAB_ANNOTATIONS);
+    });
+
+    it('sets the selectedTab to page if only page notes are present', function () {
```

Comment:
I think you meant `sets the selectedTab to "notes"` or `sets the selectedTab to "TAB_NOTES"`?